packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -353,6 +353,7 @@ function useBlockEditorSettings( settings, postType, postId, renderingMode ) {
 		setIsInserterOpened,
 		sectionRootClientId,
 		globalStylesData,
+		globalStylesLinksData,
```

Comment:
Discussion of whether this variable needs to be renamed, or alternatives to the delivery of global styles data to the block editor via settings: https://github.com/WordPress/gutenberg/pull/60100#discussion_r1687165325